app/src/main/java/org/gdg/frisbee/android/activity/MainActivity.java

```diff
@@ -379,7 +380,11 @@ public int getItemPosition(Object object) {
 
         @Override
         public int getCount() {
-            return 3;
+            if (App.getInstance().isOrganizer()) {
+                return 4;

```

Comment:
While we are doing this, I think we can also get rid of this hard coded numbers. We can have an array to store the name of the tabs, and count would be the size of that array.